### PR TITLE
Triage score changes

### PR DESCRIPTION
**Pull request by sfradkin**
Some refactoring to pull scoring into the TriageTask class, updating scoring rules so that issues overdue have increasing score values

### CODE REVIEW

File: fc/jira/tasks.py [hunk 1]
```diff
@@ -1,124 +1,35 @@
...
-    'Medium': 5,
-    'medium': 5,
-    'Low': 10,
-    'low': 10
-}
+def triage_search(auth: Auth):
+    triage_arr = []
+    triage_issues = _search_for_triage(auth)
+    for issue in triage_issues['issues']:
+        print('Retrieving info for issue: {}'.format(issue['key']))
```
Comment (halprin) [inline fc/jira/tasks.py]: Debug output?  Can we remove?

File: fc/jira/tasks.py [hunk 2]
```diff
@@ -1,124 +1,35 @@
...
+        triage_arr.append(get_task(api_url, issue['key'], auth))
```
Comment (halprin) [inline fc/jira/tasks.py]: This method can actually be done in a much more Pythonic way.
```python
def triage_search(auth: Auth) -> List[Task]:
    return [get_task(api_url, issue['key'], auth) for issue in _search_for_triage(auth)['issues']]
```

File: fc/jira/tasks.py (hunk 1 above)
Comment (sfradkin) [reply fc/jira/tasks.py]: I actually left that in there on purpose because otherwise there's no indication of what's going on.

Comment (halprin) [reply fc/jira/tasks.py]: Ah, OK.  There is no way to do the Pythonic list comprehension with this print statement.  So, this is the way to go, but can you change the `print` statement use `click.echo` function instead.

**Pull request merged**